File: Antwort.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Quizly
{
    public class Antwort
    {
        public int Id { get; set; }
        public string Text { get; set; }
        public bool IstRichtig { get; set; }
    }
}

File: Android/app/src/main/java/com/quizly/android/data/model/Antwort.kt
package com.quizly.android.data.model

import kotlinx.serialization.Serializable
import kotlinx.serialization.SerialName

/**
 * Ported from Antwort.cs (bool IstRichtig) / antwort.py (int correct_answer, 0/1).
 * [IntOrBooleanSerializer] tolerates either shape since the live API's exact
 * representation is unconfirmed.
 */
@Serializable
data class Antwort(
    val id: Int,
    val text: String,
    @SerialName("correct_answer")
    @Serializable(with = IntOrBooleanSerializer::class)
    val isCorrect: Boolean = false
)
